src/context/Video/context.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+import { createContext, useContext, useEffect, useState } from "react";
+
+const VideoContext = createContext(null);
+
+const VideoProvider = ({ children }) => {
+  const [videosAll, setVideos] = useState([]);
+  const getAllVideos = async () => {
```

Comment:
Instead of doing this in context, you can keep the context clean and make api calls in separate file and call it here

Reply:
Thanks for pointing it out
,will make the required changes